.github/workflows/ci.yml

```diff
@@ -103,11 +101,21 @@ jobs:
     name: "Docker Arm64 build"
     runs-on: macos-11
     env:
-      SNUBA_IMAGE: ghcr.io/getsentry/snuba-ci
+      # `sentry devservices up snuba` picks up the `amd64` version of the image
+      # For now, we will publish to a different name until we want to officially
+      # support the image on Docker Hub
+      # NOTE: This image has only been tested for local development
+      SNUBA_IMAGE: ghcr.io/getsentry/snuba-arm64-dev
```

Comment:
This overwrites the top-level environment variable.

The day we want to officially support `arm64` images (publish on Docker Hub) and we want to use those for local dev, we can investigate why docker-py does not use the host's platform as the tag.

Reply:
In this PR, this is the actual fix for using the `arm64` image on Apple M1s.